src/Queue.php

```diff
@@ -94,29 +205,96 @@ public function run(Job $job) {
                 $id = $job->route;
             }
             \Yii::error("Fatal Error: Error running route '{$id}'. Message: {$e->getMessage()}", 'yii2queue');
-            throw new \yii\base\Exception("Error running route '{$id}'. Message: {$e->getMessage()}. File: {$e->getFile()}[{$e->getLine()}]. Stack Trace: {$e->getTraceAsString()}", 500);
+            throw new \yii\base\Exception(
```

Comment:
Ideally the error/exception handler is an attribute / compose-able, so people can change the handler with what they want. Maybe they don't want to have `\Yii::error`, or maybe they want specific job to have warning instead of error, or maybe they want different output format so it can easily parsed.